### PR TITLE
Add function to Reply meesage as per bot.json file

### PR DESCRIPTION
**Pull request by zeel-codder**
#325 

work: reply to the user as per bot.json also, esolve spintax (10)

Till this point, the reply in the text is done.

### CODE REVIEW

**Comment by vasani-arpit** [comment]
Don't use prettier or any other formator extension. Changes should be only what you have changed not ' to " or space related changes.

**Comment by zeel-codder** [comment]
ok

**Comment by zeel-codder** [comment]
Test all functions of reply in private as well as in group chat. know stated working on the webhook part.

File: src/index.js [hunk 1]
```diff
@@ -175,6 +182,160 @@ async function Main() {
     }
 }
 
+
+async function sendReply({ msg, client, data,noMatch }) {
+    function delay(ms) {
+        return new Promise((resolve) => {
+            setTimeout(resolve, ms);
+        });
+    }
```
Comment (vasani-arpit) [inline src/index.js:191]: Delay is already available in utils. just import it from there.

**Pull request merged**